view/adminhtml/templates/affirm/error.phtml

```diff
@@ -6,7 +6,7 @@
  */
 ?>
 <h2>
-    <?php echo __(
+    <? echo __(
```

Comment:
Is Magento enforcing us to use short tags? Wondering for compatibility noted here: https://www.php.net/manual/en/language.basic-syntax.phptags.php

```
As short tags can be disabled it is recommended to only use the normal tags (<?php ?> and <?= ?>) to maximise compatibility.
```

Reply:
Yes, we got instructed by adobe to do this